diff --git a/lib/index.js b/lib/index.js
index 581af582f326d2bf4bb12ddfcd7fd30cef7a2941..9d681f8f084c9e3fe52aeed80f80830b054d06df 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -25,15 +25,15 @@ import {
   isOneOf,
   isString,
   isUndefined,
-} from '@react-native-firebase/app/lib/common';
+} from '@react-native-firebase/app/lib/common/index.js';
 
 import {
   createModuleNamespace,
   FirebaseModule,
   getFirebaseRoot,
-} from '@react-native-firebase/app/lib/internal';
+} from '@react-native-firebase/app/lib/internal/index.js';
 import { setReactNativeModule } from '@react-native-firebase/app/lib/internal/nativeModule';
-import { isBoolean } from '@react-native-firebase/app/lib/common';
+import { isBoolean } from '@react-native-firebase/app/lib/common/index.js';
 
 import { validateStruct, validateCompound } from './struct';
 import fallBackModule from './web/RNFBAnalyticsModule';
